app/util/bitbucket/populate_db.sh

```diff
@@ -116,7 +116,21 @@ else
   echo "$(cat ${BITBUCKET_BASE_URL_FILE}) was written to the ${BITBUCKET_BASE_URL_FILE} file."
 fi
 
-echo "Step4: Download DB dump"
+echo "Step4: Write license to file"
+BITBUCKET_LICENSE_FILE="license"
+if [[ -s ${BITBUCKET_LICENSE_FILE} ]]; then
+  echo "File ${BITBUCKET_LICENSE_FILE} was found. License: $(cat ${BITBUCKET_LICENSE_FILE})."
+else
+  PGPASSWORD=${BITBUCKET_DB_PASS} psql -h ${DB_HOST} -d ${BITBUCKET_DB_NAME} -U ${BITBUCKET_DB_USER} -tAc \
+  "select prop_value from app_property where prop_key = 'license';" | sed "s/\r//g" > ${BITBUCKET_LICENSE_FILE}
```

Comment:
why do you need sed "s/\r//g" ?

Reply:
We need this to remove '\r' or '^M' characters from license